src/DeviceInterface/InterfacePrototype.js

```diff
@@ -869,10 +868,11 @@ class InterfacePrototype {
     /**
      * This serves as a single place to create a default instance
      * of InterfacePrototype that can be useful in testing scenarios
+     * @param {Partial<GlobalConfig>} [globalConfigOverrides]
      * @returns {InterfacePrototype}
      */
-    static default () {
-        const globalConfig = createGlobalConfig()
+    static default (globalConfigOverrides) {
+        const globalConfig = createGlobalConfig(globalConfigOverrides)
```

Comment:
This isn't really needed. But I had to add it temporarily to test something in the test suite and thought we can leave it here in case it's useful in the future. It just enables passing specific configs during tests.